src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -0,0 +1,166 @@
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+ <template>
+    <div class="c-telemetry-view"
+         :style="styleObject">
+        <div v-if="showLabel"
+              class="c-telemetry-view__label">
+            <div class="c-telemetry-view__label-text">{{ item.domainObject.name }}</div>
+        </div>
+
+        <div v-if="showValue"
+              class="c-telemetry-view__value"
+              :class="[telemetryClass]">
+            <div class="c-telemetry-view__value-text">{{ telemetryValue }}</div>
+        </div>
+    </div>    
+ </template>
+
+<style lang="scss">
+    @import '~styles/sass-base';
+
+    .c-telemetry-view {
+        display: flex;
+        align-items: stretch;
+
+        > * {
+            // Label and value holders
+            flex: 1 1 auto;
+            display: flex;
+            flex-direction: row;
+           // justify-content: center;
+            align-items: center;
+            overflow: hidden;
+            padding: $interiorMargin;
+
+            > * {
+                // Text elements
+                @include ellipsize();
+            }
+        }
+
+        > * + * {
+            margin-left: $interiorMargin;
+        }
+
+        .c-frame & {
+            @include abs();
+            border: 1px solid transparent;
+        }
+    }
+</style>
+
+ <script>
+    export default {
+        inject: ['openmct'],
+        props: {
+            item: Object
+        },
+        computed: {
+            showLabel: function () {
+                let displayMode = this.item.config.alphanumeric.displayMode;
+                return (displayMode === 'all' || displayMode === 'label') ? true : false;
+            },
+            showValue: function () {
+                let displayMode = this.item.config.alphanumeric.displayMode;
+                return displayMode === 'value' || displayMode == 'all' ? true : false;
+            },
+            styleObject: function () {
+                let alphanumeric = this.item.config.alphanumeric;
+                return {
+                    backgroundColor: alphanumeric.fill,
+                    borderColor: alphanumeric.stroke,
+                    color: alphanumeric.color,
+                    fontSize: alphanumeric.size
+                }
+            }
+        },
+        data() {
+            return {
+                telemetryValue: '',
+                telemetryClass: ''
+            }
+        },
+        methods: {
+            getTelemetry(domainObject) {
+                return Promise.resolve(domainObject)
```

Comment:
This doesn't need to be wrapped in a promise chain, can just be done synchronously.

Reply:
Will do.